### PR TITLE
Copying and ammending registry files for FE Colleges, from alpha to beta

### PR DESCRIPTION
**Pull request by trvllngmn**
Copying registry files from Alpha to Beta

### CODE REVIEW

**Comment by Jamie-Atkinson** [comment]
approved

File: data/beta/field/region.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: curie
+field: region
+phase: beta 
+register: ''
```
Comment (gidsg) [inline data/beta/field/region.yaml]: I think this should be removed rather than empty string.

**Comment by gidsg** [comment]
@trvllngmn this register is missing a `custodian` and `register-title`. Please set these in another pull request.

**Comment by gidsg** [comment]
@trvllngmn please delete the corresponding files from Alpha rather than just copying them.

File: data/beta/field/region.yaml (hunk 1 above)
Comment (trvllngmn) [reply data/beta/field/region.yaml]: do you mean delete row 5, or delete the empty string?

Comment (gidsg) [reply data/beta/field/region.yaml]: I think delete line 5.

**Pull request merged**